insQ.js

```diff
@@ -49,16 +52,25 @@ var insertionQ = (function () {
 
         document.head.appendChild(styleAnimation);
 
-        var bindAnimationLater = setTimeout(function () {
+        const registerEventListeners = () => {
             document.addEventListener('animationstart', eventHandler, false);
             document.addEventListener('MSAnimationStart', eventHandler, false);
             document.addEventListener('webkitAnimationStart', eventHandler, false);
-            //event support is not consistent with DOM prefixes
-        }, options.timeout); //starts listening later to skip elements found on startup. this might need tweaking
+        }
+        
+        if(timeoutRequired) {
+            var bindAnimationLater = setTimeout(function () {
+                registerEventListeners();
+                //event support is not consistent with DOM prefixes
+            }, options.timeout); //starts listening later to skip elements found on startup. this might need tweaking
+        }
+        else {
+            registerEventListeners();
+        }
 
         return {
             destroy: function () {
-                clearTimeout(bindAnimationLater);
+                timeoutRequired ? clearTimeout(bindAnimationLater) : '';
```

Comment:
not a very good use of ternary operator.  You don't even care about the return here.  Just use a plain old `if`